query/src/sessions/sessions.rs

```diff
@@ -52,14 +49,8 @@ pub type SessionManagerRef = Arc<SessionManager>;
 impl SessionManager {
     pub fn from_conf(conf: Config, cluster: ClusterRef) -> Result<SessionManagerRef> {
         let catalog = Arc::new(DatabaseCatalog::try_create_with_config(conf.clone())?);
-        // Register local/system and remote database engine.
-        if conf.query.disable_local_database_engine == "0" {
-            catalog.register_db_engine("LOCAL", Arc::new(LocalDatabases::create(conf.clone())))?;
-        }
-        catalog.register_db_engine("SYSTEM", Arc::new(SystemDatabases::create(conf.clone())))?;
-        catalog.register_db_engine("REMOTE", Arc::new(RemoteDatabases::create(conf.clone())))?;
-        // Register the example for demo.
-        catalog.register_db_engine("EXAMPLE", Arc::new(ExampleDatabases::create(conf.clone())))?;
+
+        catalog.register_db_engine("example", Arc::new(ExampleDatabaseEngine::create()))?;
```

Comment:
Maybe we can remove `example` because it's based on the old design to show as an example.

Reply:
I added a new implementation for example database